modules/css-converter/docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,34 @@
+# Manual Testing — Variables First MVP
```

Comment:
I will remove the docs files before the hackathon, I don't think we need it at this stage.